.env.baklava

```diff
@@ -19,8 +19,6 @@ BLOCKSCOUT_WEB_REPLICAS=3
 # existing database.
 BLOCKSCOUT_DB_SUFFIX="1"
 BLOCKSCOUT_SUBNETWORK_NAME="Baklava"
-# Needed by blockscout to get the balances from the baklava genesis block
```

Comment:
Now `CHAIN_SPEC_PATH` is optional (if it is not present it will try to download from gpc default bucket using `BLOCKSCOUT_SUBNETWORK_NAME`). If present you can setup the url for the chainfile or only a json with the rewards as before (https://github.com/celo-org/blockscout/pull/70/files)